components/tools/ToolsSideBar.vue

```diff
@@ -1,30 +1,98 @@
 <template>
-  <nav>
-    <ul>
+  <nav :class="{'close':isClose}" class="nav height-1-1">
+    <button class="pa-3 fll hidden-xs-only hidden-lg-and-up" @click="toglleSideBar">
+      <v-icon>mdi-{{ isClose ? 'chevron-left' : 'close' }}</v-icon>
+    </button>
+    <ul class="pa-0 flc">
       <li
-        v-for="menu in menus"
-        :key="menu.link"
+        v-for="route in routes"
+        :key="route.link"
+        class="li__wrapper ovh"
       >
-        <nuxt-link :to="`/tools/${menu.link}`">
-          {{ menu.name }}
+        <nuxt-link
+          :to="`/tools/${route.path}`"
+          class="li d-flex px-4 justify-space-between"
+          :class="{'active':route.isActive}"
+        >
+          <span class="black--text"> {{ route.text }} </span>
+          <v-icon>mdi-chevron-left</v-icon>
         </nuxt-link>
       </li>
     </ul>
   </nav>
 </template>
 
 <script>
+import { getRouteName } from '~/app/utils/routeNames.ts';
+
 export default {
   name: 'ToolsSideBar',
   data: () => ({
-    menus: [
-      { name: 'שאילתות ונאומים בני דקה', link: 'one-minute-speech' },
-      { name: 'הצעות לסדר יום', link: 'genreal-agenda' },
-    ],
+    isClose: false,
   }),
+  computed: {
+    routes() {
+      const toolRoots = this.$router.options.routes.find(route => route.path === '/tools');
+      if (!toolRoots && !toolRoots.children) {
```

Comment:
I believe it should be `||` instead `&&`

Reply:
Thank you I will fix it